atlante/server/coordinator/field/status.go

```diff
@@ -0,0 +1,187 @@
+package field
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+)
+
+/*
+type Status enum {
+	Requested,
+	Started,
+	Processing {
+		Description string `json:"description"`
+	},
+	Failed {
+		Error  error `json:"error"`
+	},
+}
+*/
+
+type (
+
+	// Status is used to hold a status for searilization
+	Status struct {
+		Status StatusEnum
```

Comment:
Can we name our way around this stutter?

Reply:
Leaving in Stutter till we have a better idea. Created issue #45  for this.